docs-v2/integrations/all/accelo.mdx

```diff
@@ -1,29 +1,43 @@
 ---
 title: Accelo
 sidebarTitle: Accelo
+provider: accelo
 ---
 
-API configuration: [`accelo`](https://nango.dev/providers.yaml)
+import Overview from "/snippets/overview.mdx";
 
-## Features
+import PreBuiltTooling from "/snippets/generated/accelo/PreBuiltTooling.mdx"
+import PreBuiltUseCases from "/snippets/generated/accelo/PreBuiltUseCases.mdx"
 
-| Features | Status |
-| - | - |
-| [Auth (OAuth)](/guides/getting-started/authorize-an-api-from-your-app) | ✅ |
-| [Sync data](/guides/getting-started/read-from-an-api) | ✅ |
-| [Perform workflows](/guides/getting-started/write-to-an-api) | ✅ |
-| [Proxy requests](/guides/proxy-requests-to-an-api) | ✅ |
-| [Receive webhooks](/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+<Overview />
+<PreBuiltTooling />
+<PreBuiltUseCases />
 
-<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+## Access requirements
```

Comment:
I feel like this section is really not clear. I thought it was "Accessing this integration on Nango" and it took me while to get that it's "can you create an app in the third party?" 

Not sure how to improve, maybe some introduction text, removing the confusing `?`, changing the label names or removing the section altogether 😅

Reply:
I'll wait to hear from @bastienbeurier on this one.

Reply:
This section needs to be backfilled manually anyway, so we have time to iterate on it. Currently, it's an empty state, so inherently insufficient.

Reply:
maybe it can just be left blank when not relevant? Bad messaging is worse than no messaging imo